web_server/server.js

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (c) 2017 The WebRTC project authors. All Rights Reserved.
```

Comment:
2017 -> 2019

Reply:
It's a clone from github.com/webrtc/test - I thought about making it a separate commit, but then forgot about it.